bot/cogs/information.py

```diff
@@ -1,13 +1,18 @@
 import colorsys
 import logging
+import pprint
 import textwrap
 import typing
+from typing import Any, Mapping, Optional
 
-from discord import CategoryChannel, Colour, Embed, Member, Role, TextChannel, VoiceChannel, utils
-from discord.ext.commands import Bot, Cog, Context, command
+import discord
+from discord import CategoryChannel, Colour, Embed, Member, TextChannel, VoiceChannel
+from discord import Role, utils
```

Comment:
Any reason why these lines were split?

Reply:
The only reason I can think about is the limitation of 120 characters, but I did not count

Reply:
no, I'm guessing PyCharm had a moment